apiserver/pkg/model/converter_test.go

```diff
@@ -568,6 +572,21 @@ func TestAutoscalerOptions(t *testing.T) {
 	assert.Len(t, options.Volumes, 2)
 }
 
+func TestNilAutoscalerOptions(t *testing.T) {
+	options := convertAutoscalingOptions(nil)
+	assert.Nil(t, options)
+}
+
+func TestFromCrdToAPIClusters(t *testing.T) {
```

Comment:
Could you please help pre-allocate here?
https://github.com/ray-project/kuberay/blob/747708b7cc123eb270ada8f84dbc4a988836db03/apiserver/pkg/model/converter.go#L92

Reply:
Just to clarify, pre-allocation means `apiClusters := make([]*api.Cluster, 0, N) `